src/main/java/org/jitsi/jigasi/JvbConference.java

```diff
@@ -1221,6 +1221,30 @@ else if(ChatRoomMemberPresenceChangeEvent.MEMBER_UPDATED
                             + " " + member.getContactAddress());
 
             jigasiChatRoomMembers.remove(member.getName());
+
+            CallPeer peer;
+            if (jvbCall != null && (peer = jvbCall.getCallPeers().next()) instanceof MediaAwareCallPeer)
+            {
+                MediaAwareCallPeer peerMedia = (MediaAwareCallPeer) peer;
+                peerMedia.getConferenceMembers().stream().forEach(m -> {
+
+                    String address = ((ConferenceMember)m).getAddress();
```

Comment:
Is the cast to `ConferenceMember` necessary?

Reply:
Yep
```
                  /jigasi/src/main/java/org/jitsi/jigasi/JvbConference.java:[1231,39] cannot find symbol
[ERROR]   symbol:   method getAddress()
[ERROR]   location: variable m of type java.lang.Object
```